### ISSUE TITLE
v2.7.8 BUG: HEBI Core Library not found

### ISSUE THREAD

**Issue by w3ichen** [HebiRobotics/hebi-python-examples#76]
When I installed `hebi-py` version 2.7.8 (the newest version released in July 16, 2024), the `import hebi` caused this error below. 
I'm running on linux x86_64 machine, ubuntu.
I tried installed the older 2.7.6 version and there's no issues.
I found this [old stack overflow thread ](https://stackoverflow.com/questions/55827053/setup-hebi-robot-on-python)talking about this issue and it being related to 32-bit machines but I'm on a 64-bit machine.

```bash
[HEBI Warning] Unable to load (library=hebi).
[HEBI Warning] Candidate libraries attempted (in order):
[HEBI Warning] Candidate 1: /home/user/.local/lib/python3.8/site-packages/hebi/lib/linux_x86_64/libhebi.so.2.14
[HEBI Warning] Candidate 2: /home/user/.local/lib/python3.8/site-packages/hebi/lib/linux_x86_64/libhebi.so.2
[HEBI Warning] Candidate 3: /home/user/.local/lib/python3.8/site-packages/hebi/lib/linux_x86_64/libhebi.so
Traceback (most recent call last):
  File "/home/user/catkin_ws/src/environmental_robot/arm_control/scripts/arm_control.py", line 7, in <module>
    import hebi
  File "/home/user/.local/lib/python3.8/site-packages/hebi/__init__.py", line 10, in <module>
    from ._internal.ffi import loader as __loader
  File "/home/user/.local/lib/python3.8/site-packages/hebi/_internal/ffi/__init__.py", line 1, in <module>
    from .loader import _handle
  File "/home/user/.local/lib/python3.8/site-packages/hebi/_internal/ffi/loader.py", line 240, in <module>
    _handle = _init_libraries()
  File "/home/user/.local/lib/python3.8/site-packages/hebi/_internal/ffi/loader.py", line 234, in _init_libraries
    core_lib, core_loc = _load_core_shared_library()
  File "/home/user/.local/lib/python3.8/site-packages/hebi/_internal/ffi/loader.py", line 173, in _load_core_shared_library
    def _load_core_shared_library(): return _load_shared_library('hebi', 'HEBI Core', 'HEBI_C_LIB')
  File "/home/user/.local/lib/python3.8/site-packages/hebi/_internal/ffi/loader.py", line 167, in _load_shared_library
    raise RuntimeError('{} library not found'.format(readable_name))
RuntimeError: HEBI Core library not found
```

**Comment by cwbollinger** [comment]
Hi Weichen,

Can you confirm which version of Ubuntu you're using? There's a known issue where hebi-py 2.7.8 does not work on Ubuntu 20.04 machines due to an issue we're having with glibc symbol versions (see [changelog])

**Comment by w3ichen** [comment]
@cwbollinger yep, i'm on ubuntu 20.04. Is it possible to make `pip install hebi-py` install the older compatible version if it's being installed on a ubuntu 20.04 machine?

**Comment by cwbollinger** [comment]
Yes, you should be able to use this syntax to stay on `2.7.6`: 

```
# uninstall existing version of hebi-py
pip uninstall hebi-py
# now install the desired version (2.7.6 is the latest version without the symbol issue)
pip install -v hebi-py==2.7.6
```

I will leave this issue open in case others have this same question, and close it once we've released a new version that fixes the issue for Ubuntu 20.04 users.

**Comment by cwbollinger** [comment]
Closing this issue, as Ubuntu 20.04 is usable again with the latest Python version.
